_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,12 @@
+---
+question: "What is a cEP and why it is so important to do analysis pre-gsoc and design as early as possible, to be finalized in the cEP?"
+---
+A **cEP (coala Enhancement Proposal)** is proposed to the coala community to discuss the new major features that the contributor intends to implement during GSoC.
+
+**cEP's** are designed before implementation. Implementing major features usually happens from a technical perspective and may lead to poorly designed content - developer and user wise. This can essentially save a lot of coding work and also makes sure that major features are properly documented.
```

Comment:
Shouldn't it be `cEP`, instead of `cEP's`?

Reply:
Used `cEP's`  beacause maybe user is interested in submitting more than one proposals.

Reply:
or we can use cEP(s) !!

Reply:
Yeah, but the plural of cEP should be cEPs, right?
That is, coala Enhancement Proposals.

Reply:
yep, right

Reply:
so should I replace `cEP's` with `cEPs` ?